src/components/chips/js/chipsDirective.js

```diff
@@ -62,6 +62,8 @@
    *  </span>
    *
    * @param {string=|object=} ng-model A model to bind the list of items to
+   * @param {expression=} md-on-chip-select An expression to be run each time a new item got selected
```

Comment:
I thought, these expressions will be called after a ``chip` gets selected, so I wan't to find a clear attribute for the API (and `md-on-select` can be interpreted as select event for the input field)